concepticondata/conceptrelations.tsv

```diff
@@ -770,3 +770,12 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3805	SHY OR ASHAMED	narrower	1894	ASHAMED
 3809	SCREAM (PRODUCE A CRY)	producedby	2522	SCREAM (NOISE)
 298	SHRIEK	similar	3809	SCREAM (PRODUCE A CRY)
+3213	HIRE OR RENT	narrower	3831	RENT (SOMETHING)
+947	STREAM	narrower	3821	BROOK
+1208	CAT	narrower	3825	MALE CAT
+1936	DEER	instanceof	3826	ROE DEER
+832	BEAN	instanceof	3839	BROAD BEAN
+3830	WALL	narrower	933	WALL (OF HOUSE)
+1123	PREGNANT	narrower	3827	PREGNANT (OF ANIMALS)
```

Comment:
Is it `narrower` or `instance of`?

Reply:
I can see the point, but with adjectives, that's hard to decide...

Reply:
Here I would really go for narrower. Instanceof sounds like animal pregnancy is one of different types of pregnany when it might just be human convention to have separate words for humans and other mammals (with the biological background being pretty much the same).